instances/widgets.treasury-factory.near/widget/components/AccountInput.jsx

```diff
@@ -51,6 +52,10 @@ useEffect(() => {
 }, [isValidAccount]);
 
 const checkAccountAvailability = async () => {
+  // skip check if it's implicit account
+  if (allowNonExistentImplicit && (account ?? "").length === 64) {
```

Comment:
Do we also want to check that it's a hex string? only the characters 0-9a-f?

Reply:
good point, I will add it.